src/main/java/org/embulk/input/marketo/delegate/ProgramInputPlugin.java

```diff
@@ -134,14 +138,20 @@ public TaskReport ingestServiceData(PluginTask task, RecordImporter recordImport
     {
         // query by date range and incremental import and not preview
         if (task.getQueryBy().isPresent() && task.getQueryBy().get() == QueryBy.DATE_RANGE && task.getIncremental() && !Exec.isPreview()) {
-            DateTime latestUpdateAt = new DateTime(task.getLatestUpdatedAt().get());
-            DateTime now = DateTime.now();
+            OffsetDateTime latestUpdateAt = task.getLatestUpdatedAt().isPresent() ?
```

Comment:
No need to check the LatestUpdateAt again because the validation already checked the present of this value